### PR TITLE
used log of lifetime

### LINKED ISSUE #30: Adapt scaling of diagrams
https://github.com/philippabele/deep-groove-with-pytorch/blob/main/deep_groove_project/src/first_eda/first_eda.py#L43

The parameters speed and force are linear distributed but the lifetime looks more like a log suited metrics. May you could take the log of the Lifetime in order to provide Logarithmic scaling?

I think, that your charts will be more informative with this adaption.

### PR DESCRIPTION
**Pull request by RicoKursidem**
resolves #30 
I used the log of Lifetime in the second graph. 
Should I also use it in the first one, or is it good enough to see the correlations?

### CODE REVIEW

File: deep_groove_project/src/first_eda/first_eda.py [hunk 1]
```diff
@@ -42,10 +43,10 @@ def show_3d_scatter(df):
     n = df.iloc[:, 1]
     lifetime = df.iloc[:, 2]
 
-    ax.scatter(fr, n, lifetime)
+    ax.scatter(fr, n, np.log(lifetime))
```
Comment (philippabele) [inline deep_groove_project/src/first_eda/first_eda.py]: I would like to use the log with base 10 here

https://numpy.org/doc/stable/reference/generated/numpy.log10.html#numpy.log10

The natural logarithm does not provide a good base from a engineering perspective.

File: deep_groove_project/src/first_eda/first_eda.py [hunk 2]
```diff
@@ -42,10 +43,10 @@ def show_3d_scatter(df):
...
     ax.set_xlabel('Fr')
     ax.set_ylabel('n')
-    ax.set_zlabel('Lifetime')
+    ax.set_zlabel('log(Lifetime)')
```
Comment (philippabele) [inline deep_groove_project/src/first_eda/first_eda.py]: It is good practice to provide the phyiscal units. "Lifetime in h"

Comment (philippabele) [inline deep_groove_project/src/first_eda/first_eda.py]: Please use the log with base 10 for all representations of the lifetime.

**Review by philippabele — CHANGES_REQUESTED** [review]

File: deep_groove_project/src/first_eda/first_eda.py (hunk 1 above)
Comment (RicoKursidem) [reply deep_groove_project/src/first_eda/first_eda.py]: changes it to log10(Lifetime)

File: deep_groove_project/src/first_eda/first_eda.py (hunk 2 above)
Comment (RicoKursidem) [reply deep_groove_project/src/first_eda/first_eda.py]: added the value to the Label

Comment (RicoKursidem) [reply deep_groove_project/src/first_eda/first_eda.py]: changed the function to log10( )

Comment (RicoKursidem) [reply deep_groove_project/src/first_eda/first_eda.py]: I also provided physical units for the other parameters

**Review by philippabele — APPROVED** [review]

**Pull request merged**